server/controllers/ctrlHelper.js

```diff
@@ -0,0 +1,7 @@
+const parseOrSetEmpty = (value, language, setEmpty = false) => {
```

Comment:
And actually this function might not be needed at all.

Isn't it more clear to just write for example
```courseCredits: ladokCourseCredits ?? ''```
instead of
```courseCredits: parseOrSetEmpty(ladokCourseCredits)```

The `parseOrSetEmpty` in kursinfo-web defaults to "No information inserted"/"Ingen information tillagd", so it makes more sense to have it there (even though I suspect that we use it to often there as well)

Reply:
I agree with you so I will make the required changes 🙌